content/en/blog/2025/otel-day.md

```diff
@@ -0,0 +1,196 @@
+---
+title:
+  'OTel Community Day + Open Observability Summit: One Big Day for Open Source
+  Observability'
+linkTitle: OTel Community Day
+date: 2025-06-13
+author: '[Isabella Langan](https://github.com/isabella-rose-l) (Causely)'
+# prettier-ignore
+cspell:ignore: Langan Causely Horovits Alok Bhide Radlein Kuncham Canuel Danielson OllyGarden Vashistha Kumar Baylis Roshore Budhaditya Bhattacharya Ravindra Bhargava Distrogen Braydon Kains Mansi Agrawal Todea Aircall Liudmila Molkova Cijo Jernigan Spec-tacular Helmuth Krietz Prasad Mujumdar Okahu Vijay Heffner Huxing Zhang Minghui
```

Comment:
Spelling is still in error. It might be easier to just turn off the spell checking in the schedule part.